src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -348,6 +348,8 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ngraph::Func
         {opset4::GreaterEqual::get_type_info_static(), fuse_type_to_binary_comparision<opset4::GreaterEqual>},
         {opset4::Less::get_type_info_static(), fuse_type_to_binary_comparision<opset4::Less>},
         {opset4::LessEqual::get_type_info_static(), fuse_type_to_binary_comparision<opset4::LessEqual>},
+        {ov::opset10::IsFinite::get_type_info_static(), fuse_type_to_binary_comparision<ov::opset10::IsFinite>},
```

Comment:
BTW, need to consider usage of CC factory here, because in case of conditional compilation, this map will hold all the operations, which are not even used in the model.
See https://github.com/openvinotoolkit/openvino/blob/master/src/common/conditional_compilation/include/openvino/cc/factory.h

Reply:
@ilya-lavrenov any idea about how to handle such problem? We can see this table contains quite a lot op items, not all will be used in each model, we also need consider them. Anyhow, we need a solution to solve this issue.

Reply:
As I wrote - use CC factory

Reply:
Agree, we need consider CC issue for this type_to_fuse map. Let's create a ticket to track it and maybe a separated PR.

Reply:
Create a ticket: https://jira.devtools.intel.com/browse/CVS-114864